Doc/includes/email-headers.py

```diff
@@ -1,5 +1,6 @@
 # Import the email modules we'll need
-from email.parser import BytesParser, Parser
+# from email.parser import BytesParser
```

Comment:
I'd say move this line to line 6 and change the comment to "[...] these three lines".

Reply:
Done